docs/api/js/accounts-l1-l2.md

```diff
@@ -228,20 +239,21 @@ const abi = [
 ];
 const contractInterface = new ethers.utils.Interface(abi);
 const calldata = contractInterface.encodeFunctionData("increment", []);
-const l2gasLimit = BigNumber.from(1000);
+const l2GasLimit = BigNumber.from(1000);
 
 const txCostPrice = await wallet.getBaseCost({
   gasPrice,
   calldataLength: ethers.utils.arrayify(calldata).length,
-  l2gasLimit,
+  l2GasLimit,
 });
 
 console.log(`Executing the transaction will cost ${ethers.utils.formatEther(txCostPrice)} ETH`);
 
 const executeTx = await wallet.requestExecute({
-  calldata,
-  l2gasLimit,
   contractAddress: "0x19a5bfcbe15f98aa073b9f81b58466521479df8d",
+  calldata,
+  l2Value: 1,
```

Comment:
why do we supply 1 here?

Reply:
@AnastasiiaVashchuk any reason why?

Reply:
@StanislavBreadless  randomly, or I understand the mission of this variable wrong?

Reply:
@StanislavBreadless  there is also contract address value... For me looks random - there is no explanation for what is it

Reply:
@AnastasiiaVashchuk `l2Value` is a field which denotes how much ETH to you want to be present within this L1->L2 transaction

Reply:
so if it is an example why we can not just pass `l2Value` = 1?

Reply:
@AnastasiiaVashchuk we can, but then we should explain what we are doing here. It is just in the previous example no value was passed